src/modules/registration/components/Registration.js

```diff
@@ -38,6 +38,7 @@ export const Registration = () => {
                     email: data['email'],
                     token: response.data.token
                 }
+                localStorage.setItem('userToken', user.token)
```

Comment:
same iof this is in setUser then we dont need it in both componenets

Reply:
done